R/targets/flu_data_targets.R

```diff
@@ -122,28 +123,29 @@ create_flu_data_targets <- function() {
           wait_seconds = 1,
           fn = pub_covidcast,
           source = "nssp",
-          signal = "pct_ed_visits_influenza",
+          signals = "pct_ed_visits_influenza",
           time_type = "week",
           geo_type = "state",
           geo_values = "*",
           fetch_args = g_fetch_args
         )
-        nssp_hhs <- retry_fn(
-          max_attempts = 10,
-          wait_seconds = 1,
-          fn = pub_covidcast,
-          source = "nssp",
-          signal = "pct_ed_visits_influenza",
-          time_type = "week",
-          geo_type = "hhs",
-          geo_values = "*",
-          fetch_args = g_fetch_args
-        )
+        # nssp_hhs <- retry_fn(
+        #   max_attempts = 10,
+        #   wait_seconds = 1,
+        #   fn = pub_covidcast,
+        #   source = "nssp",
+        #   signals = "pct_ed_visits_influenza",
+        #   time_type = "week",
+        #   geo_type = "hhs",
+        #   geo_values = "*",
+        #   fetch_args = g_fetch_args
+        # )
```

Comment:
when you were talking about hhs regions, I thought you meant for nhsn, ILI, and flusurv. Those are definitely a net negative.

As exogenous features hhs regions were a bit more ambiguous and are probably worth keeping around.